dylan-craig/styles/layout.css

```diff
@@ -0,0 +1,106 @@
+#logo {
+  float: left;
+  padding-bottom: 25px;
+  color: black;
+  text-decoration: none;
+}
+
```

Comment:
This **#logo img** would probably fall more in line with a module since it points to a more specific element within a layout class. Same goes for the **#logo h1**, the **.print img**, the **.recipe li**, etc... One way to look at it is as you go from base, layout, and module the styling gets more specific. Starting with very generalized styling at the base and getting down to very specific styles in modules.